### DISCUSSION TITLE
Add data in ArchR enrichment Bulk ATAC-seq

### DISCUSSION THREAD

**Discussion by NoemieL** [GreenleafLab/ArchR#1172] [Feature Requests]
Hi,
I am wondering if it will be possible to add in the ArchR enrichment bulk ATAC-seq the ATAC-seq data from this paper https://ashpublications.org/blood/article/131/19/2138/36919/Active-enhancer-and-chromatin-accessibility ?

Otherwise is there a way to do it myself without to have to download manually all the data from the original ArchR enrichment as proposed in the manual section 12.4 Custom Enrichment?

Thank you very much in advances

**Comment by rcorces** [comment]
> I am wondering if it will be possible to add in the ArchR enrichment bulk ATAC-seq the ATAC-seq data from this paper https://ashpublications.org/blood/article/131/19/2138/36919/Active-enhancer-and-chromatin-accessibility ?

If you are asking whether we (ArchR devs) can add this to some centralized place for your use, no we cannot do that.

> Otherwise is there a way to do it myself without to have to download manually all the data from the original ArchR enrichment as proposed in the manual section 12.4 Custom Enrichment?

I'm not sure what you mean but this seems like precisely the use case described in Chapter 12.4. You need to have bed regions that you obtain from whatever data you want. That is all.

**Reply by NoemieL** [reply]
In fact I want to add to the ArchR bulk ATAC the bed regions from this paper to add info of cells (plasma cells) that are not on your database (this could also interest other people that's why I was asking). If you are not interested or can't do it, it is not a problem. I also want to know if there is an easy way to add this bed file to your data: how can I get your all bulk ATAC dataset without to do it manually (as Chapter 12.4) for each subset of your dataset? Then if I could get easily your dataset, I will add the extra bed regions to it based on Chapter 12.4. I am just looking to a way to add extra bed to your existing bulk ATAC dataset for the enrichment analysis.

**Reply by rcorces** [reply]
Perhaps you are referring to the files listed within the `addArchRAnnotations()` function:
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/AnnotationPeaks.R#L653-L663

**Reply by NoemieL** [reply]
Maybe it is this.
Is it related to the info I fund on the annotation peaks summary?
ATAC_summary=readRDS(file = "/Annotations/ATAC-In-Peaks-Summary.rds")

colData(ATAC_summary$regionMatches) #info data
DataFrame with 96 rows and 2 columns
                                          paper             paperTitle
                                        <array>                <array>
Brain_Astrocytes         https://www.biorxiv... Single-cell epigenom..
Brain_Excitatory_neurons https://www.biorxiv... Single-cell epigenom..
Brain_Inhibitory_neurons https://www.biorxiv... Single-cell epigenom..
Brain_Microglia          https://www.biorxiv... Single-cell epigenom..
Brain_Oligodendrocytes   https://www.biorxiv... Single-cell epigenom..
...                                         ...                    ...
Heme_LMPP                https://www.nature.c.. Lineage-specific and..
Heme_MEP                 https://www.nature.c.. Lineage-specific and..
Heme_Mono                https://www.nature.c.. Lineage-specific and..
Heme_MPP                 https://www.nature.c.. Lineage-specific and..
Heme_NK                  https://www.nature.c.. Lineage-specific and..
